### PR TITLE
Add Norwegian provider

### PR DESCRIPTION
**Pull request by phaza**
The personal identity number has the correct number of digits, but will not be valid because of check sum error.
Hopefully someone else will have time to look at it.

### CODE REVIEW

File: src/Faker/Provider/no_NO/Address.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
..."Arendal", "Askim", "Bergen", "Bodø", "Brekstad", "Brevik", "Brumunddal", "Bryne", "Brønnøysund", "Drammen", "Drøbak", "Egersund", "Elverum", "Fagernes", "Farsund", "Fauske", "Finnsnes", "Flekkefjord", "Florø", "Fosnavåg", "Fredrikstad", "Førde", "Gjøvik", "Grimstad", "Halden", "Hamar", "Hammerfest", "Harstad", "Haugesund", "Hokksund", "Holmestrand", "Honningsvåg", "Horten", "Hønefoss", "Jessheim", "Jørpeland", "Kirkenes", "Kolvereid", "Kongsberg", "Kongsvinger", "Kopervik", "Kragerø", "Kristiansand", "Kristiansund", "Langesund", "Larvik", "Leknes", "Levanger", "Lillehammer", "Lillesand", "Lillestrøm", "Lyngdal", "Mandal", "Mo i Rana",  "Moelv", "Molde", "Mosjøen", "Moss", "Mysen", "Måløy", "Namsos", "Narvik", "Notodden", "Odda", "Orkanger", "Oslo", "Otta", "Porsgrunn", "Risør", "Rjukan", "Røros", "Sandefjord", "Sandnes", "Sandnessjøen", "Sandvika", "Sarpsborg", "Sauda", "Ski", "Skien", "Skudeneshavn", "Sortland", "Stathelle", "Stavanger", "Stavern", "Steinkjer", "Stjørdalshalsen", "Stokmarknes", "Stord", "Svelvik", "Svolvær", "Tromsø", "Trondheim", "Tvedestrand", "Tønsberg", "Ulsteinvik", "Vadsø", "Vardø", "Verdalsøra", "Vinstra", "Åkrehamn", "Ålesund", "Åndalsnes", "Åsgårdstrand"
```
Comment (fzaninotto) [inline src/Faker/Provider/no_NO/Address.php]: Could you split that to several lines? Same for other very long lines. Eases review.

File: src/Faker/Provider/no_NO/Person.php [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
...amara", "Tania", "Tanja", "Tanya", "Tara", "Taran", "Tatiana", "Tea", "Terese", "Thale", "Thanh", "Thao", "Thea", "Thelma", "Theodora", "Therese", "Thi", "Thilde", "Thina", "Thine", "Thora", "Thorbjørg", "Thordis", "Thorild", "Thu", "Thuy", "Thyra", "Tia", "Tiffany", "Tilde", "Tina", "Tindra", "Tine", "Tiril", "Toini", "Tomine", "Tone", "Tonje", "Tora", "Torbjørg", "Tordis", "Torgny", "Torgun", "Torgunn", "Torhild", "Tori", "Toril", "Torild", "Torlaug", "Torny", "Torunn", "Tove", "Toya", "Trine", "Trude", "Turid", "Tuva", "Tuyet", "Tyra", "Ulla", "Ulrikke", "Una", "Undis", "Une", "Unn", "Unni", "Ursula", "Uzma", "Valentina", "Vanessa", "Vanja", "Vebjørg", "Velaug", "Venche", "Vendela", "Vera", "Veronica", "Veslemøy", "Vibeke", "Victoria", "Vida", "Vigdis", "Viktoria", "Vilde", "Vilhelmina", "Vilja", "Villemo", "Vilma", "Viola", "Virginia", "Vivi", "Vivian", "Vår", "Vårin", "Wanda", "Wanja", "Wenche", "Wendy", "Wera", "Weronika", "Wibecke", "Wibeke", "Wigdis", "Wilde", "Wilma", "Winnie", "Xuan", "Yasmin", "Ylva", "Yngvild", "Yvonne", "Zahida", "Zahra", "Zainab", "Zara", "Zuzanna", "Øydis", "Øyvor", "Ågot", "Aasa", "Aase", "Åse", "Åshild", "Aashild", "Åslaug", "Åsne", "Åsta", "Aasta"
```
Comment (fzaninotto) [inline src/Faker/Provider/no_NO/Person.php]: Could you split by initial? Same for other arrays in this provider.

Comment (phaza) [inline src/Faker/Provider/no_NO/Person.php]: As in split by first character?

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**